### ISSUE TITLE
Wondering about the need to unregister known accessories

### ISSUE THREAD

**Issue by christophevg** [homebridge-nhc2/homebridge-nhc2#21]
https://github.com/wvanvlaenderen/homebridge-nhc2/blob/371a7d519884c4c295347cd0fded05317fa29567/src/nhc2-platform.ts#L139-L141

@wvanvlaenderen (I'm going to test it, but maybe you can confirm it faster from previous experience ;-)) I think that unregistering known accessories causes them to be removed from HomeKit, and afterwards being re-added, _with new defaults_ like their room causing them to "overwrite" configuration in the Home app. Was there any important reason to do it this way? And not simply skip them when they are known?

**Comment by MirandaPoth** [comment]
I wondered about this too. I think it might be needed when a device changes its name (I had that myself this morning) but otherwise I'm not sure. I think it might make sense to only do it if the name changes?

**Comment by christophevg** [comment]
Good point, and if that's the case _and_ it indeed is the cause of my accessories returning to their default room now and then, we will have to see which changes are to be pushed through on the same uuid and how to do this without unregistering them.
First going to try and finish [sun blinds support](https://github.com/wvanvlaenderen/homebridge-nhc2/tree/feature/sunblinds) 😇

**Comment by wvanvlaenderen** [comment]
@christophevg I've added this because of warnings on startup during initial development of the add-on. I assume, as @MirandaPoth suggested, only unregistering on specific changes should be the preffered setting.
